src/qt/res/README.md

```diff
@@ -0,0 +1,46 @@
+# Icon policy
+This document provides guidelines for contributing icons to Bitcoin Core.
+
+## Preparing Icons
+Both an icon source file, in Scalable Vector Graphics (SVG) format, and an optimized production file, in Portable Network Graphics (PNG) format, are required for each icon.
+
+#### SVG Source File
+SVGs are used as source files because they can scale while retaining image quality but are not used in production due to limited application support.
+If different-sized production (PNG) icons are required, they can be generated from the associated SVG source file in any vector-based tool.
+
+#### PNG Production File
+PNGs are used in production due to wide application support, transparency support, and better image quality compared to competing file types
+such as JPEG.
+
+### Icon Grid
+Bitcoin Core uses an 8-point grid system for its icons which utilizes size increments of 8px to visually orientate elements within a grid. The majority of screen sizes are divisible by 8, which makes fitting elements easier using an 8-point grid system. This system makes things more efficient for designers and aids in maintaining consistency across applications.
+
+### Icon Style
+Icon style should adhere to the following principles to maintain consistency and clarity:
+
+- To ensure icons look good at any scale, a minimalistic design should be used.  
+- Icon size should be 32x32px by default. A different sized icon may be necessary under special circumstances. If the 32x32px size is too small for the required icon placement, then the icon should be scaled up or down while adhering to the 8-point grid system.
+- The icon should avoid organic shapes or elements, these do not scale well.
+- Colors should be consistent with existing Bitcoin Core icons.
+- Icons should adhere to the 8-point grid system (see above).
+

```

Comment:
```suggestion
### Icons
Bitcoin Core primarily uses icons from the [Bitcoin Icon set](https://github.com/BitcoinDesign/Bitcoin-Icons). Please follow the design guidelines laid out by that project when designing icons.

[Bitcoin icons](https://bitcoinicons.com/) is an open source icon set made for bitcoin applications.

```

Reply:
Well not yet. But, we can add that in when we attempt a migration to those icons

Reply:
Should I just remove the `Icon Style` section for now?

Reply:
I think we should merge this a long with the updated icons from Bitcoin Icons

Reply:
alright, sounds good. Made draft already.